.github/workflows/cabal.yml

```diff
@@ -29,32 +29,41 @@ jobs:
     - name: 📥 Checkout repository
       uses: actions/checkout@v1
 
+    - name: 🧰 Select optimal cabal version
+      run: |
+        case "$OS" in
+          Windows_NT)   echo "CABAL_VERSION=3.4.0.0-rc5"  >> $GITHUB_ENV;;
+          *)            echo "CABAL_VERSION=3.4.0.0-rc4"  >> $GITHUB_ENV;;
```

Comment:
I should probably re-upload rc4 to the webhost.haskell.org server, because this will still stress-test olegs private server: https://github.com/haskell/actions/issues/15#issuecomment-753708623

Reply:
Yes please :D